src/libs/ReportUtils.ts

```diff
@@ -4181,6 +4181,11 @@ function getReportPreviewMessage(
         let actualPayerName = report.managerID === currentUserAccountID ? '' : getDisplayNameForParticipant({accountID: report.managerID, shouldUseShortForm: true});
         actualPayerName = actualPayerName && isForListPreview && !isPreviewMessageForParentChatReport ? `${actualPayerName}:` : actualPayerName;
         const payerDisplayName = isPreviewMessageForParentChatReport ? payerName : actualPayerName;
+        if (isForListPreview && report?.parentReportID != null) {
+            const parentReportSearchAll = getReport(report?.parentReportID, allReports);
+            const payerText = isDM(parentReportSearchAll) && translatePhraseKey === 'iou.payerPaidAmount' ? '' : payerDisplayName;
```

Comment:
We already have an existing `parentReport` variable, so we don't need to create a new one.
```suggestion
            const payerText = isDM(parentReport) && translatePhraseKey === 'iou.payerPaidAmount' ? '' : payerDisplayName;
```